server/index.js

```diff
@@ -7,10 +7,16 @@ const path = require('path');
 
 app.use(bodyParser.json());
 
-app.use(express.static(path.join(__dirname, '../public')))
+app.use('/', express.static('./public/'));
+app.use(/\/\d+\//, express.static('./public/'));
 
-app.get('/api/about', (req, res) => {
-  db.About.findOne()
+
+app.get('/api/about:id', (req, res) => {
+  var id = req.params.id.split(':');
```

Comment:
probably related to my above comment.. but splitting params is generally unnecessary/undesired